plugin/machine-learning-config.conf

```diff
@@ -0,0 +1,67 @@
+# ------------------------------------------------------------------------
+# OWASP ModSecurity Core Rule Set Plugin
+# Copyright (c) 2021-2022 Core Rule Set project. All rights reserved.
+#
+# The OWASP ModSecurity Core Rule Set plugins are distributed under
+# Apache Software License (ASL) version 2
+# Please see the enclosed LICENSE file for full details.
+# ------------------------------------------------------------------------
+
+# OWASP CRS Plugin
+# Plugin name: machine-learning
+# Plugin description: This plugin serves to add a machine learning integration support to the CRS.
+# Rule ID block base: 9,516,000 - 9,516,999
+# Plugin version: 1.0.0
+
+# Documentation can be found here:
+# https://github.com/coreruleset/machine-learning-integration-plugin
+
+# Generic rule to disable the plugin
+#
+# Plugins are enabled by default.
+#
+# They become active by placing them in the plugin folder. It is possible to
+# control plugin activation via setting a variable. This can be done in the
+# plugin config file here.
+#
+# The predefined variable name is meant to be "<plugin name>-plugin_enabled".
+# For the machine-learning-plugin, this means it can be disabled by setting
+# tx.machine-learning-plugin_enabled=0.
+#
+# Note that a global setting of this variable overrides the setting here.
+# That means the "enabled" variable is only set by this rule if it has not
+# been set before.
+#
+# Feel free to set the variable unconditionally here by replacing the
+# SecRule line with an unconditional SecAction statement.
+#
+#SecRule &TX:machine-learning-plugin_enabled "@eq 0" \
+#  "id:9516010,\
+#   phase:1,\
+#   pass,\
+#   nolog,\
+#   setvar:'tx.machine-learning-plugin_enabled=0'"
+
+# The Machine Learning plugin can be configured to run in 2 modes.
+# machine-learning-plugin_mode=1 means it will run in "false positive detection mode. 
+# This mode will help reduce the number of false positives.
+# machine-learning-plugin_mode=2 means it will run in "detection and training mode". 
+# inbound_ml_status by default is set to 0. 
+# This will be updated by the Machine Learning model and then compared to the pass or deny flag.
+# inbound_ml_pass_flag = 1 is initialized against which the decision to pass is made.
+# inbound_ml_deny_flag = 0 is initialized against which the decision to deny is made.
+
+SecAction \
+ "id:9516020,\
+  phase:1,\
+  nolog,\
+  pass,\
+  t:none,\
+  ver:'machine-learning-plugin/1.0.0',\
+  setvar:'tx.machine-learning-plugin_mode=2',\
+  setvar:'tx.inbound_ml_status=0',\
+  setvar:'tx.inbound_ml_pass_flag=1',\
+  setvar:'tx.inbound_ml_deny_flag=0',\
+  setvar:'tx.inbound_ml_anomaly_score=0',\
+  setvar:'tx.inbound_ml_threshold=0', \
+  setvar:'tx.ml_server_url=http://127.0.0.1:5000/'"
```

Comment:
I suggest to unite variable naming. The best would be to keep format used by `tx.machine-learning-plugin_mode` variable i.e. prefix names with `tx.machine-learning-plugin_`.